Mathlib/MeasureTheory/Constructions/Convolution.lean

```diff
@@ -0,0 +1,115 @@
+/-
+Copyright (c) 2023 Josha Dekker. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Josha Dekker
+-/
+import Mathlib.MeasureTheory.Constructions.Prod.Basic
+import Mathlib.MeasureTheory.Measure.MeasureSpace
+/-!
+# The multiplicative and additive convolution of measures
+
+In this file we define and prove properties about the convolutions of two measures
```

Comment:
```suggestion
In this file we define and prove properties about the convolutions of two measures.
```

Reply:
Can you please explain in this file docstring what the convolution is? State that it's the map of + or * on the product.